llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -480,22 +500,20 @@ def dtensor_to_tensor_hook(
 
             log.debug(f'Creating new model instance')
 
-            if composer_model.using_peft:
-                # We don't use meta here because the state dict does not contain the full
```

Comment:
want to point out that this comment is wrong -- the state dict does contain the full model in addition to the adapter weights.

This is important & needed if we want to merge the adapters later on. So it's a good thing lol